hashicorp/terraform-provider-tfe
# [TF-10830] add new attribute to org resource and data blocks

## Description

Adding new organization attribute, `aggregated_commit_status_enabled` to organization data/resource blocks.

_Remember to:_


## Testing plan

1. Create a `tfe_organization` resource with the required arguments: `name`, `email`.
1. Add the new `aggregated_commit_status_enabled` argument,  then and apply the changes
1. Verify against target TF host that is has been updated to reflect the assigned value: true, false

Example config:
```tf
resource "tfe_organization" "my_org" {
  email = "testemail@test.com"
  name = "test-org"
  aggregated_commit_status_enabled = true
}
```

## External links

_Include any links here that might be helpful for people reviewing your PR. If there are none, feel free to delete this section._

- [go-tfe documentation](https://pkg.go.dev/github.com/hashicorp/go-tfe#Organization)
- [API documentation](https://developer.hashicorp.com/terraform/cloud-docs/api-docs/organizations#sample-response)
- [Related PR](https://github.com/hashicorp/go-tfe/pull/829)

## Output from acceptance tests

_Please run applicable acceptance tests locally and include the output here. See [testing.md](https://github.com/hashicorp/terraform-provider-tfe/blob/main/docs/testing.md) to learn how to run acceptance tests._

_If you are an external contributor, your contribution(s) will first be reviewed before running them against the project's CI pipeline._

```
$ TESTARGS="-run TestAccTFEOrganization_update_costEstimation" make testacc                            
TF_ACC=1 TF_LOG_SDK_PROTO=OFF go test $(go list ./... |grep -v 'vendor') -v -run TestAccTFEOrganization_update_costEstimation -timeout 15m
?       github.com/hashicorp/terraform-provider-tfe     [no test files]
testing: warning: no tests to run
PASS
ok      github.com/hashicorp/terraform-provider-tfe/internal/client     0.227s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/hashicorp/terraform-provider-tfe/internal/logging    0.306s [no tests to run]
?       github.com/hashicorp/terraform-provider-tfe/version     [no test files]
=== RUN   TestAccTFEOrganization_update_costEstimation
    resource_tfe_organization_test.go:118: Skipping this test until the SDK can support importing resources before applying a configuration
--- SKIP: TestAccTFEOrganization_update_costEstimation (0.00s)
PASS
ok      github.com/hashicorp/terraform-provider-tfe/internal/provider   0.530s
```

```
TESTARGS="-run TestAccTFEOrganization_update_costEstimation" make testacc
TF_ACC=1 TF_LOG_SDK_PROTO=OFF go test $(go list ./... |grep -v 'vendor') -v -run TestAccTFEOrganization_update_costEstimation -timeout 15m
?       github.com/hashicorp/terraform-provider-tfe     [no test files]
testing: warning: no tests to run
PASS
ok      github.com/hashicorp/terraform-provider-tfe/internal/client     0.314s [no tests to run]
testing: warning: no tests to run
PASS
ok      github.com/hashicorp/terraform-provider-tfe/internal/logging    0.150s [no tests to run]
?       github.com/hashicorp/terraform-provider-tfe/version     [no test files]
=== RUN   TestAccTFEOrganization_update_costEstimation
    resource_tfe_organization_test.go:118: Skipping this test until the SDK can support importing resources before applying a configuration
--- SKIP: TestAccTFEOrganization_update_costEstimation (0.00s)
PASS
ok      github.com/hashicorp/terraform-provider-tfe/internal/provider   0.538s
```
